components/text/the_silver_searcher/Makefile

```diff
@@ -0,0 +1,43 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL)". You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright (c) 2016 Sergey Avseyev
+#
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=		the_silver_searcher
+COMPONENT_VERSION=	0.31.0
+COMPONENT_PROJECT_URL=	http://geoff.greer.fm/ag/
+COMPONENT_SUMMARY=	A code searching tool similar to ack, with a focus on speed.
+COMPONENT_FMRI=         text/the_silver_searcher
```

Comment:
I prefer the name author gives it, moreover Fedora also use it: https://apps.fedoraproject.org/packages/the_silver_searcher